File: matrices.py
def main():
    personas = []
    while input("¿Desea ingresar una persona? (s/n): ").lower() == 's':
        nombre = input("Ingrese el nombre: ")
        apellido = input("Ingrese el apellido: ")
        dni = input("Ingrese el DNI: ")
        # Se solicitan los teléfonos separados por coma
        telefonos = [tel.strip() for tel in input("Ingrese los teléfonos separados por coma: ").split(',')]
        personas.append([nombre, apellido, dni, telefonos])
    
    print("\nDatos ingresados:")
    for persona in personas:
        print(persona)

if __name__ == "__main__":
    main()

File: matrices.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function ask(question) {
  return new Promise(resolve => rl.question(question, resolve));
}

async function main() {
  const personas = [];
  while ((await ask("¿Desea ingresar una persona? (s/n): ")).toLowerCase() === 's') {
    const nombre = await ask("Ingrese el nombre: ");
    const apellido = await ask("Ingrese el apellido: ");
    const dni = await ask("Ingrese el DNI: ");
    // Se solicitan los teléfonos separados por coma
    const telefonos = (await ask("Ingrese los teléfonos separados por coma: ")).split(",").map(t => t.trim());
    personas.push([nombre, apellido, dni, telefonos]);
  }
  
  console.log("\nDatos ingresados:");
  personas.forEach(persona => console.log(persona));
  rl.close();
}

main();